src/Parameter.hpp

```diff
@@ -29,7 +28,7 @@ namespace imu_aceinna_openimu {
         {"periodic-packet-rate", 4, PARAM_INTEGER, nullptr},
         {"acceleration-filter", 5, PARAM_INTEGER, nullptr},
         {"angular-velocity-filter", 6, PARAM_INTEGER, nullptr},
-        {"orientation", 7, PARAM_ORIENTATION, nullptr},
+        {"orientation", 7, PARAM_STRING, nullptr},
```

Comment:
Is this orientation regarding the magnetometer axis swap thing? A doc would be nice for new users

Reply:
Yes. I was thinking the same ... was not 100% sure it works in the first place.